pkg/handlers/v1/notification.go

```diff
@@ -56,6 +59,7 @@ func (h *NotificationHandler) Handle(ctx context.Context) (Output, error) {
 
 	scanNotifications := make([]scanNotification, len(scans))
 	for offset, scan := range scans {
+		scanCompletedTime := scan.Timestamp
```

Comment:
nit: This assignment is not necessary. You could also do `time.Since(scan.Timestamp)` below and it would work the same.

Reply:
done!